docs/api/data_collector.md

```diff
@@ -22,6 +22,7 @@ data_collector_callbacks/episode_metadata_callback
 .. autofunction:: minari.DataCollectorV0.step
 .. autofunction:: minari.DataCollectorV0.reset
 .. autofunction:: minari.DataCollectorV0.close
+.. autofunction:: minari.DataCollectorV0.create_dataset
 .. autofunction:: minari.DataCollectorV0.save_to_disk
```

Comment:
Can you already `save_to_disk` from doc?

Reply:
done